dagmc_stats/DagmcStats.py

```diff
@@ -6,7 +6,7 @@
 
 class DagmcStats:
 
-    def __init__(self, filename, populate=False):
+    def __init__(self, filename, populate=False, meshset=None):
```

Comment:
let's not pass in a meshset- we don't load the file yet so it doesn't make sense to be able to pass this.